src/apps/code-editor/src/app/components/FileDrawer/FileDrawer.js

```diff
@@ -104,6 +110,15 @@ export const FileDrawer = memo(function FileDrawer(props) {
         {open ? null : (
           <span className={styles.bodyText}>More file information</span>
         )}
+        {open ? (
+          <span className={cx(styles.bodyText, styles.DrawerShortcut)}>
+            ({platform.isMac ? "CMD" : "CTRL"} + D) Close Drawer
```

Comment:
We could move the logic of `platform.isMac ? "CMD" : "CTRL"` into the hook and have it return that value so users of the hook do not need to handle this aspect. It would be a little awkward from a function signature and return value perspective but the utility is very convenient.